ko/admin/control.rst

```diff
@@ -1302,7 +1302,7 @@ QUERY_EDITOR 브로커는 다음과 같은 응용의 접속 요청만을 허용
 
 ::
 
-    broker_changer    broker_name [cas_id] parameters value
+    broker_changer    broker_name [cas_id] conf_name conf_value
```

Comment:
When broker_changer run, it's usage is the following.
broker_changer <broker-name> [<cas-number>] <conf-name> <conf-value>
So, I think that you would better to change this to the printed usage.